backend/src/v2/metadata/client.go

```diff
@@ -281,6 +281,14 @@ func (e *Execution) FingerPrint() string {
 	return e.execution.GetCustomProperties()[keyCacheFingerPrint].GetStringValue()
 }
 
+func UpdateTaskNameWithDagID(taskName string, dagID int64) string {
```

Comment:
This function name is misleading in my opinion as it makes it sound like something in MLMD is getting updated. How about just removing `Update` or replace with `Get`.